zebra-state/src/service/finalized_state.rs

```diff
@@ -17,18 +17,18 @@ use self::disk_format::{DiskDeserialize, DiskSerialize, FromDisk, IntoDisk, Tran
 
 use super::QueuedBlock;
 
-/// The finalized part of the chain state, stored in sled.
+/// The finalized part of the chain state, stored in the db.
 ///
 /// This structure has two categories of methods:
 ///
-/// - *synchronous* methods that perform writes to the sled state;
+/// - *synchronous* methods that perform writes to the db state;
 /// - *asynchronous* methods that perform reads.
 ///
 /// For more on this distinction, see RFC5. The synchronous methods are
 /// implemented as ordinary methods on the [`FinalizedState`]. The asynchronous
 /// methods are not implemented using `async fn`, but using normal methods that
 /// return `impl Future<Output = ...>`. This allows them to move data (e.g.,
-/// clones of handles for [`sled::Tree`]s) into the futures they return.
+/// clones of handles for the database) into the futures they return.
```

Comment:
I think this comment isn't accurate any more, maybe we should remove it entirely?